### DISCUSSION TITLE
target directory of a maven project is not included

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [mfussenegger/nvim-jdtls#612] [Q&A]
Hi All,

I have a maven Spring GraphQL project where the GQL entities are generated. The generated code under the `${project.dir}/target/generated-sources/graphql` is not picked up. NeoVim says "cannot find symbol". Any idea how can I tell JDTLS that the above mentioned directory should be included?

Thanks!

**Comment by Andras-Csanyi** [comment]
I asked the [same question](https://stackoverflow.com/questions/77914223/how-to-configure-eclipses-language-server-to-include-additional-source-director) on SO with further details and an example project.

**Comment by mfussenegger** [comment]
See the discussion in https://github.com/redhat-developer/vscode-java/issues/177

Depending on how the sources are generated, eclipse.jdt.ls might pick it up (e.g. antlr maven plugins work fine). Others might require extra configuration.

**Comment by mbert** [comment]
I am having the same issue. OpenAPI source generator, project generated for eclipse, `.classpath` looks good (source folder added), but getting error messages in nvim. Since I installed everything into a vanilla nvim installation, I guess there still is a problem with the default setup.

If there is a workaround that people could use in the meantime, it would be great to have it documented somewhere where people can find it. So far I haven't been successful to fix it for myself.
